alibi_detect/cd/pytorch/learned_kernel.py

```diff
@@ -35,6 +35,7 @@ def __init__(
             batch_size: int = 32,
             preprocess_batch_fn: Optional[Callable] = None,
             epochs: int = 3,
+            num_workers: int = 4,
```

Comment:
Any reason for 4 as default `num_workers`?

Reply:
Only to match what you had in the KeOps one:

https://github.com/SeldonIO/alibi-detect/blob/master/alibi_detect/cd/keops/learned_kernel.py#L38

Reply:
Well that backfired quickly! Let me check why I did that.

Reply:
I believe I used it as `num_workers = 4 * num_gpus` looks like a reasonable default. For KeOps we could assume (with a bit of a stretch) that people use it with GPU (which is the whole point). However this is less the case for the normal PyTorch one. Maybe we should change them all to the PyTorch dataloader's default 0?

Reply:
Good idea, I'll change default to `0` for all.

Reply:
Updated default to `num_workers=0` in ffe1b78